PyInstaller/utils/hooks/__init__.py

```diff
@@ -492,7 +492,10 @@ class method (e.g., `idontevenknow<1.6,>1.9,!=1.9.6,<2.0a0,==2.4c1`) is
         version = get_module_attribute(module_name, version_attr)
 
     # Compare this version against the version parsed from these requirements.
-    return version in requirements_parsed
+    if version is '':
```

Comment:
I suggest simply checking for `not version` which will match even if `get_module_attribute()` would return `None`.

Also this should go in front of the comment and get a comment of its own like "module does not exist".